app/user/rest.py

```diff
@@ -135,8 +137,8 @@ def send_user_code(user_id):
     create_user_code(user, secret_code, verify_code.get('code_type'))
```

Comment:
Do you want to move this into the task? Guess we could do it here - and the application fails and the user knows? Also not a performance bottleneck on our application?